spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,12 @@
+# Preview all emails at http://localhost:3000/rails/mailers/outbound_email_mailer
```

Comment:
I think this URL is specific to `OutboundEmailMailer` - if so, perhaps we need to change the URL here?

Reply:
Ah, good catch.